packages/component-submission/server/aggregates/Submission.test.js

```diff
@@ -1,6 +1,8 @@
 const Manuscript = require('@elifesciences/component-model-manuscript').model
 const File = require('@elifesciences/component-model-file').model
 const Team = require('@elifesciences/component-model-team').model
+const SemanticExtraction = require('@elifesciences/component-model-semantic-extraction')
```

Comment:
need to change package to export on `model` like the others

Reply:
This isn't the case in tests like this as the `@pubsweet/model` package registers the models at server start up which doesn't happen when running unit tests.